influence_benchmark/environment/environment.py

```diff
@@ -1,61 +1,133 @@
 import random
-from typing import Dict, Tuple, Any, List
-import openai
+
+import yaml
+
+from influence_benchmark.environment.character import character
+from influence_benchmark.environment.state import State
+from influence_benchmark.environment.transition_model import TransitionModel
 
 
 class Environment:
-    def __init__(self, config, env_name: str):
+    def __init__(self, config: dict):
         self.config = config
-        self.state = None
-        self.env_name = env_name
-
-    def reset(self):
-        self.state = self.generate_initial_state()
+        self.env_name = config["env_name"]
+        self.backend = config["env_backend_model"]
+        self.transition_model = TransitionModel(self.env_name, "gpt-4o")
+        with open(f"influence_benchmark/config/env_configs/{self.env_name}.yaml", "r") as file:
```

Comment:
We should probably handle data better at some point, rather than using relative imports. If this ends up causing problems (as it usually does), lmk and I can tell you better ways to handle it @carolius @weissercn